modules/auxiliary/scanner/mysql/mysql_login.rb

```diff
@@ -33,7 +33,8 @@ def initialize(info = {})
 
     register_options(
       [
-        Opt::Proxies
+        Opt::Proxies,
+        OptBool.new('CreateSession', [false, 'Create a new session for every successful login', false])
       ])
 
     options_to_deregister = %w[PASSWORD_SPRAY]
```

Comment:
Since we're here, let's add the notification message similar to postgres

```
    options_to_deregister = %w[SQL PASSWORD_SPRAY]
    if framework.features.enabled?(Msf::FeatureManager::POSTGRESQL_SESSION_TYPE)
      add_info('New in Metasploit 6.4 - The %grnCreateSession%clr option within this module can open an interactive session')
    else
      options_to_deregister << 'CreateSession'
    end
    deregister_options(*options_to_deregister)
```

Reply:
Sure 👍